tests/test_telegramobject.py

```diff
@@ -255,6 +257,17 @@ def test_pickle_apply_api_kwargs(self, bot):
         assert obj.foo == "bar"
         assert obj.api_kwargs == {}
 
+    def test_pickle_removed_attribute(self):
+        """Test when newer versions of the library remove attributes from classes (which the old
+        pickled versions still have).
+        """
+        # load old Chat object, which doesn't have api_kwargs attribute, and has now removed attr
+        # `all_members_are_administrators`.
+        pp = PicklePersistence(data_file("old_chat.pickle"))
+        chat = pp._load_file(data_file("old_chat.pickle"))
```

Comment:
I would prefer it if we didn't have to rely on protected methods for the tests … maybe you can create the pickle file via `PicklePersistence` itself? It would also be nice to have a few CSI-style comments about how the file was created such that one can reproduce it if necessary :)